samples/CombinedSkillsSamples/cs/DetectAndTrackObjectsSample_UWP/MainPage.xaml

```diff
@@ -0,0 +1,78 @@
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    mc:Ignorable="d"
+    Background="{ThemeResource ApplicationPageBackgroundThemeBrush}"
+    Loaded="Page_Loaded">
+
+    <Grid>
+        <Grid.RowDefinitions>
+            <RowDefinition Height="1*" />
+            <RowDefinition Height="10*" />
+        </Grid.RowDefinitions>
+        <ScrollViewer Grid.Row="0" Grid.RowSpan="2">
+            <StackPanel>
+                <controls:Expander VerticalAlignment="Top"
+                               Header="Details and options.."
+                               HorizontalContentAlignment="Stretch"
+                               Foreground="Black"
+                               Background="#FF7F7F7F"
+                               BorderBrush="Black"
+                               IsExpanded="False"
+                               ExpandDirection="Down"
+                               Expanded="UIExpander_Expanded"
+                               Collapsed="UIExpander_Expanded">
+                    <StackPanel Padding="5">
+                        <TextBlock Text="Skill Descriptions:" FontWeight="Bold"/>
+                        <TextBox Name="UISkillDescription" IsReadOnly="True" AcceptsReturn="True"/>
+                        <TextBlock Text="Available ObjectDetector execution devices:" FontWeight="Bold"/>
+                        <ListBox Name="UIDetectorExecutionDevices" SelectionChanged="UIDetectorExecutionDevices_SelectionChanged"/>
+                        <TextBlock Text="Available ObjectTracker execution devices:" FontWeight="Bold"/>
+                        <ListBox Name="UITrackerExecutionDevices" SelectionChanged="UITrackerExecutionDevices_SelectionChanged"/>
+                        <controls:Expander VerticalAlignment="Top"
+                               Header="Object Detector ObjectKind Filters"
+                               HorizontalContentAlignment="Stretch"
+                               Foreground="Black"
+                               Background="#FF7F7F7F"
+                               BorderBrush="Black"
+                               BorderThickness="1"
+                               Margin="0,5"
+                               IsExpanded="False"
+                               ExpandDirection="Down"
+                               Expanded="UIExpander_Expanded"
+                               Collapsed="UIExpander_Expanded">
+                            <ListView Name="UIObjectKindFilters" SelectionMode="Multiple" ScrollViewer.VerticalScrollBarVisibility="Visible" SelectionChanged="UIObjectKindFilters_SelectionChanged" />
+                        </controls:Expander>
+                        <StackPanel Orientation="Horizontal">
+                            <TextBox Width="200" Margin="0,0,5,0" Header="Detector Evaluation Interval" TextChanging="ValidateNumericalTextBoxEntry" Name="UIDetectorIntervalBox" />
```

Comment:
>ValidateNumericalTextBoxEntry [](start = 118, length = 29)

suggestion: use the Community toolkit extensions for specifying regex validation on textbox entry: [https://docs.microsoft.com/en-us/windows/communitytoolkit/extensions/textboxregex](https://docs.microsoft.com/en-us/windows/communitytoolkit/extensions/textboxregex)